app/views/users/registrations/edit.html.erb

```diff
@@ -33,11 +33,6 @@
         <%= f.label :password_confirmation %><br />
         <%= f.password_field :password_confirmation, autocomplete: "off", required: false, class: "form-control", placeholder: "Confirm new password" %>
       </div>
-      
-      <div class="form-group">
-        <%= f.label :current_password %><br />
-        <%= f.password_field :current_password, hint: "we need your current password to confirm your changes", required: true, class: "form-control", placeholder: "Enter current password" %>
-      </div>
```

Comment:
@ianroberts131 Recommend keeping the current/old password requirement when a form allows changing a password. This stops an attacker who takes control of a user's session from locking that user out of their own account.